learn/what_is_meilisearch/comparison_to_alternatives.mdx

```diff
@@ -95,7 +95,7 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | Placeholder search | ✅ | ✅ | ✅ | ✅ |
-| Multi-index search (Federated search) | ✅ | ✅ | ✅ | ✅ |
+| Multi-search (Federated search) | ✅ | ✅ | ✅ | ✅ |
```

Comment:
Think this should stay the same as it changes the meaning of the comparison (and as discussed, federated search = multi-index search)

```suggestion
| Multi-index search (Federated search) | ✅ | ✅ | ✅ | ✅ |
```